lib/bap_sema/bap_sema_lift.ml

```diff
@@ -158,12 +156,20 @@ let is_conditional_jump jmp =
   Insn.(may affect_control_flow) jmp &&
   has_jump_under_condition (Insn.bil jmp)
 
-let blk cfg block : blk term list =
+let find_dst block symtab =
+  if Insn.(is call) (Block.terminator block) then
+    Symtab.find_call symtab (Block.addr block) |>
```

Comment:
A call usually refers to a call site, i.e., a place where the call occurs, e.g., the `call` assembly instruction, the `call` term in BIR, or a IRL call. In your case, the call is the Block.terminator which is a call. 
What you're finding is called _callee_, I believe, though it is hard to understand, because there is no documentation for the new Symtab functions. And they are very ambiguous and easy to use incorrectly.